project1/test.sh

```
@@ -0,0 +1,164 @@
+	expected_output=$GEN_DIR/${file_stem}.exp
+	actual_output=$GEN_DIR/${file_stem}.out
+
+    compiler_output=compiler_output.txt
+	touch $compiler_output
+	fpc $GEN_DIR/$filename &> $compiler_output
+
+	binary=$GEN_DIR/$file_stem
+	compiled=false
+	if [ -f $binary ]; then
+	    compiled=true
+	    ./$binary &> $expected_output
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)